conf/distro/openxt-test.conf

```diff
@@ -0,0 +1,21 @@
+#
+# This is how bitbake will establish its configuration:
+#
+# require conf/abi_version.conf
+# include conf/site.conf
+# include conf/auto.conf
+# include conf/local.conf
+# include conf/build/${BUILD_SYS}.conf
+# include conf/target/${TARGET_SYS}.conf
+# include conf/machine/${MACHINE}.conf
+# include conf/machine-sdk/${SDKMACHINE}.conf
+# include conf/distro/${DISTRO}.conf
+# include conf/distro/defaultsetup.conf
+# include conf/documentation.conf
+# include conf/licenses.conf
+# require conf/sanity.conf
+#
+
+DISTRO_FEATURES_append += "bats-auto"
+
+require openxt-common.inc
```

Comment:
Could you leave openxt-main as-is and just do:
```
require conf/distro/openxt-main.conf

DISTRO_FEATURES += "bats-auto"
```
?

Reply:
That's possible, I am not entirely sure what logic made me split it in a `-common`. Maybe it should be renamed `openxt-main-test`, since it will make the naming consistent should another `DISTRO` emerge.

Reply:
Either way you go with populating this file, it looks like ordering the `require` line before doing the append to `DISTRO_FEATURES` (as @jandryuk has it) is the right way around to have the lines.

Reply:
I would disagree on keeping openxt-main as I am looking at the possibility of having to introducing a new distro type that would inherit most of the `-common` and overriding what is needed to meet the need for the new distro type.